README.md

```diff
@@ -16,7 +16,7 @@ This library is made available under the [MIT](LICENSE) open source license.
 
 ## Important note
```

Comment:
```suggestion
## API stability
```
Happy to not go with this, but in my eyes it's no longer an "important note", so we probably should rename the title?